torchgen/dest/register_dispatch_key.py

```diff
@@ -58,6 +59,8 @@ def gen_registration_headers(
             headers.append("#include <ATen/cuda/EmptyTensor.h>")
     elif backend_index.dispatch_key == DispatchKey.MPS:
         headers.append("#include <ATen/mps/EmptyTensor.h>")
+    elif backend_index.dispatch_key == DispatchKey.XPU:
+        headers.append("#include <aten/EmptyTensor.h>")
```

Comment:
This file is not built-in pytorch? Should we add more comments?

Reply:
sure, I have added comment to refer the function and location, thank.